namespaces/live-1.cloud-platform.service.justice.gov.uk/sentence-planning-preprod/03-resourcequota.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: ResourceQuota
+metadata:
+  name: namespace-quota
+  namespace: sentence-planning-preprod
+spec:
+  hard:
+    requests.cpu: 3000m
+    requests.memory: 6Gi
+    limits.cpu: 6000m
```

Comment:
Please remove limits.cpu and limits.memory - they have no useful effect.

Reply:
done to both preprod and dev